srv/relayer.go

```diff
@@ -120,15 +122,39 @@ func (r *Relayer) SendUserOperation() modules.BatchHandlerFunc {
 			attribute.Int64("wait_timeout", int64(r.waitTimeout)),
 		)
 
+		var computeOPHashes = func(err error, hash common.Hash) {
+
+			for _, op := range ctx.Batch {
+
+				currentOpHash, unsolvedOpHash := utils.GetUserOpHash(op, r.ep, r.chainID)
+
+				r.m.Compute(unsolvedOpHash, func(oldValue OpHashes, loaded bool) (newValue OpHashes, delete bool) {
+
+					var mergedErr error
+					if err != nil {
+						mergedErr = multierror.Append(oldValue.Error, err)
```

Comment:
optional you can eliminate a dep by using [errors join](https://pkg.go.dev/errors#Join)